.github/workflows/record-snapshots.yml

```
@@ -14,6 +14,10 @@ on:
         required: false
         default: false
 
+permissions:
+  contents: read
+    actions: write
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)